src/common/config/client/db/configmanager.go

```diff
@@ -0,0 +1,98 @@
+package db
+
+import (
+	"fmt"
+
+	"github.com/goharbor/harbor/src/common/config"
+	"github.com/goharbor/harbor/src/common/dao"
+	"github.com/goharbor/harbor/src/common/utils/log"
+)
+
+// CoreConfigManager ... Wrap the configure driver to previous interface, used for remove adminserver container only
+type CoreConfigManager struct {
+	Driver *ConfigureDriver
+}
+
+// NewCoreConfigManager ...
+func NewCoreConfigManager() *CoreConfigManager {
+	return &CoreConfigManager{Driver: GetConfigureDriverInstance()}
+}
+
+// NewCoreConfigManagerFromArray ...
+func NewCoreConfigManagerFromArray(items []config.Item) *CoreConfigManager {
+	return &CoreConfigManager{Driver: NewDBConfigureStoreFromArray(items)}
```

Comment:
the DB driver is hardcoded here, how do we extend the driver? If the contributor wants to contribute a redis driver, the current structure seems not support well.